config.go

```diff
@@ -199,6 +199,10 @@ const (
 	// defaultCoinSelectionStrategy is the coin selection strategy that is
 	// used by default to fund transactions.
 	defaultCoinSelectionStrategy = "largest"
+
+	// defaultKeepFailedPaymentAttempts is the default setting for whether to keep
```

Comment:
nit: line too long. https://github.com/lightningnetwork/lnd/pull/6681 should help devs make this easier to avoid by auto-setting displayed tab with to 8 spaces and adding a ruler at 80 chars.

Reply:
Done.